.github/workflows/msbuild.yml

```diff
@@ -1,24 +1,50 @@
 name: Windows Build
 
 on:
-  push:
-    branches:
-      - master
-      - development
-
   workflow_dispatch:
+    inputs:
+      upload_artefacts:
+        description: "Upload build artefacts"
+        type: boolean
+        required: false
+        default: false
+      build_configuration: # "Final"
```

Comment:
Junk comment?

Reply:
Resolved in be1e4ed7c8a7bf801ee9da0edd37f25a0e28c13e